CMSgov/design-system
# Migrated guidance from the DSM to the docs site for M.gov

## Summary

[WNMGDS-1810 - Migrate Mgov DSM content into doc site](https://jira.cms.gov/browse/WNMGDS-1810)

Migrated content from the DSM to the Docs site using guidance created in this [Mural board](https://app.mural.co/t/cmsoc4625/m/cmsoc4625/1657895242729/a8e0dfa85767f39610169548df0f586845ce13c3?sender=ucf13c21d112482a4c5766770).

The following pages were not migrated as they need further instruction. 
- Spacing
- Alerts
- Layout grid

### Added

- Exceptions page to the guidelines section. 

### Changed

Added M.gov guidance to the following pages:
- Foundations
   - Typography
- Components
   - Icons
   - Buttons
   - Tooltips
   - Help drawer

Note: Transfer of guidance is not line for line from the DSM. Some of the guidance did not need to be migrated.

## How to test

- Build doc site locally
- Switch to M.gov theme
- Navigate to pages above
- Verify with DSM pages [found here](https://cms.invisionapp.com/dsm/cms/mgov-design-system)
